src/CMake/nativeTests.cmake

```diff
@@ -12,11 +12,18 @@ add_test(NAME xbutil
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
 add_test(NAME xbutil2
-  COMMAND ${CMAKE_BINARY_DIR}/runtime_src/core/tools/xbutil2/xbutil2 --new scan
+  COMMAND ${CMAKE_BINARY_DIR}/runtime_src/core/tools/xbutil2/xbutil2 --new examine
```

Comment:
We should consider removing the nativeTests.cmake.

Instead, the individual unit tests would reside reside in the application's directory.   Look at xclbinutil's unit tests for an example, where:
1. The unit tests are in the "xclbinutil/unittests" directory.
2. They are added to the CMake testing harness via the applications CMakeLists.txt file.  The cmake function to add them is xrt_add_test().

Reply:
Certainly we should move to that model in a future revision.